admin/acceptance.py

```diff
@@ -106,6 +106,13 @@ def start_nodes(self):
             cwd=self.vagrant_path.path,
             env=extend_environ(FLOCKER_BOX_VERSION=box_version))
 
+        from flocker.provision._install import run, task_pull_docker_images
+        for node in self.NODE_ADDRESSES:
+            run(
+                username='root',
+                address=node,
+                commands=task_pull_docker_images()
```

Comment:
This, and I guess other uses of ``task_pull_docker_images``, differ from the docs. In the documented example of how to move data, at http://doc-dev.clusterhq.com/gettingstarted/tutorial/volumes.html, the images are not pulled.

If we are only guaranteeing (via tests) that these commands work if `docker pull` is used first, then the docs should have those pull commands. If the docs don't say to pull the images first then another option is to increase the test timeout.